packages/devtools-components/src/tree.js

```diff
@@ -341,7 +341,9 @@ class Tree extends Component {
       // Additional classes to add to the root element.
       className: PropTypes.string,
       // style object to be applied to the root element.
-      style: PropTypes.object
+      style: PropTypes.object,
+      // Prevents blur when Tree loses focus
+      preventBlur: PropTypes.bool
```

Comment:
Maybe add a default prop value for this? You're just checking the negation so not having one should be fine but it's good to have a default.